apstools/devices/measComp_usb_ctr_support.py

```diff
@@ -120,13 +120,19 @@ class MeasCompCtr(Device):
     # https://github.com/epics-modules/measComp/blob/master/measCompApp/Db/measCompDevice.template
     model_name = Component(EpicsSignalRO, "ModelName", kind="config", string=True)
     model_number = Component(EpicsSignalRO, "ModelNumber", kind="config")
-    firmware_version = Component(EpicsSignalRO, "FirmwareVersion", kind="config", string=True)
+    firmware_version = Component(
```

Comment:
I'm puzzled.  This line is less than the project configurations: https://github.com/BCDA-APS/apstools/blob/a165d24b2ae272ba0db3fb73d9feba4416f40631/.ruff.toml#L31-L32   https://github.com/BCDA-APS/apstools/blob/a165d24b2ae272ba0db3fb73d9feba4416f40631/setup.cfg#L11-L12 https://github.com/BCDA-APS/apstools/blob/a165d24b2ae272ba0db3fb73d9feba4416f40631/pyproject.toml#L5-L6

Reply:
The project configurations are so that we do not have this kind of reformatting.

Reply:
Don't know why, but fixed it